### PR TITLE
adds some alt attributes

### LINKED ISSUE #85: add alt attribute to images
image elements do not have alt attributes
>Informative elements should aim for short, descriptive alternate text

https://dequeuniversity.com/rules/axe/2.2/image-alt?application=lighthouse

### PR DESCRIPTION
**Pull request by BO41**
closes #85

### CODE REVIEW

File: _layouts/post.html [hunk 1]
```diff
@@ -20,7 +20,7 @@ <h3 class="post-title" itemprop="name headline">{{ page.title | escape }}</h3>
         <div class="row">
             <div class="col-md-3 col-img">
                 {% assign image = site.data.images[page.image] %}
-                <img src="/img/{{ image.url }}" class="img-responsive postImg" />
+                <img src="/img/{{ image.url }}" class="img-responsive postImg" alt="post image" />
```
Comment (TobiGr) [inline _layouts/post.html]: This could be a bit more specific. E.g use
```Liquid
{% if image.name %}{{ image.name }}{% else %}post image{% endif %}
```

Comment (BO41) [reply _layouts/post.html]: yes, thanks. will do :+1:

**Comment by TobiGr** [comment]
Thanks. I just realized that we can add [here](https://github.com/TeamNewPipe/website/pull/93/files#diff-4ea7a794e4d7b7503ac520689daeb20bR16) also some more info. Fixed in https://github.com/TeamNewPipe/website/commit/4fb46d33230901819577bae06e1ab34ae5b1d5e8

**Comment by TheAssassin** [comment]
Going to deploy ASAP.

**Pull request merged**